packages/@glimmer/interfaces/lib/template.d.ts

```diff
@@ -120,7 +120,7 @@ export interface BlockMetadata {
   symbols: BlockSymbolNames;
   scopeValues: unknown[] | null;
   isStrictMode: boolean;
-  moduleName: string;
+  moduleName: string | undefined;
```

Comment:
This is actually allowed to be `undefined`, and when the auto-fixer tried to assume otherwise, it
produced failing code.